minio/console
# Add custom registry on tenant creation

fixes https://github.com/minio/console/issues/422
Step for adding custom registry when a custom image is selected was added.
<img width="879" alt="Screen Shot 2020-11-30 at 9 43 05 PM" src="https://user-images.githubusercontent.com/11819101/100697671-43a7a480-335c-11eb-8ed5-0aad3f114f33.png">

Test steps:
When creating a tenant on a k8s env set a custom image and set a custom private registry and make sure the tenant gets created as well as the secrets containing the docker credentials.
